documentation/devonfw-ide-introduction.asciidoc

```diff
@@ -1,12 +1,69 @@
-= Introduction
+= devonfw-ide
 
-http://devonfw.com[devonfw] provides a solution to building applications which combine best-in-class frameworks and libraries
-as well as industry proven practices and code conventions.
-It massively speeds up development, reduces risks and helps deliver better results.
+== What is devonfw-ide?
 
-This document contains the instructions for the tool `devonfw-ide` to set up and maintain your development tools including your favorite IDE (integrated development environment).
+`devonfw-ide` is a toolbox allowing you to pick tools which fit the requirements and needs of your project.
 
-include::features.asciidoc[leveloffset=2]
+It is not another Eclipse. It gives you Eclipse, along with Java, maven, NodeJs, VSCode and everything else you may need to quick start your development. All you need to do is download and extract a zip file and execute its setup, reducing your initial setup time for starting a project from days to minutes.
```

Comment:
It sounds like I would always get eclipse, java, maven,... You should say that you get what you want and need from that list.